build.sbt

```diff
@@ -713,36 +713,39 @@ lazy val compiler = configureAsSubproject(project)
   .dependsOn(library, reflect)
 
 lazy val interactive = configureAsSubproject(project)
+  .settings(commonSettings)
   .settings(disableDocs)
   .settings(skip in publish := true)
   .settings(
     name := "scala-compiler-interactive",
-    description := "Scala Interactive Compiler"
+    description := "Scala Interactive Compiler",
+    // TODO remove `-unit-special` once we re-STARR on 2.13.3, because scala/bug#11965
+    scalacOptions in Compile ++= Seq("-Xlint:-unit-special,_", "-unchecked", "-Werror", "-Wconf:cat=deprecation:msg=early.initializers:s"),
```

Comment:
`-Weekend-werrior`

Oh, is this the thing where patmat warnings are counted and trigger error result? On another PR, it turned out that `-Werror` means a warning was counted, not emitted as text warning. I wonder if rebase will help. That was for `-nowarn` to mean don't summarize those counts; maybe this is a similar thing.

Reply:
I've been trying to do something with `-Werror` and "werewolf", but yours is better

Sorry, I don't follow the rest of your comment, I'm confused.

Reply:
https://github.com/scala/scala/pull/8923 is the PR to make `-nowarn` not say `1 warning` when patmat warning (which is post-typer) escapes `-Wconf` suppression (at typer).

Maybe `-Werror` suffers the same issue. Just guessing.

Reply:
This PR already does have #8923 in its history.

I still don't follow whether you're giving me review feedback about something I should perhaps change in this PR, or whether your remarks are... just musings about something we should be on the lookout for.